docs/source/deployment/deploy-model-locally.rst

```diff
@@ -77,7 +77,59 @@ CSV input must be a valid pandas.DataFrame CSV representation. For example:
 JSON Input
 **********
 
-JSON input must be a dictionary with exactly one of the following fields, further specifying the input data's type and encoding:
+You can either pass a flat dictionary corresponding to the desired model payload or wrap the
+payload in a dict with a dict key that specifies your payload format. 
+
+Raw Payload Dict
```

Comment:
Can we change the order to start from the traditional wrapped input, and mention that this is special format introduced for providing compatible interface with SaaS APIs? Most of the models except OpenAI chat still need the original input format, so probably we still want to say it is "normal" and raw dict is a sort of backdoor.